lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -133,7 +136,7 @@ class FeatureCard extends React.Component {
     let newIndex = this.state.campaignIndex + 1;
 
     // Only log to GA if someone actually pressed the button
-    if (buttonTrigger) {
+    if (buttonTrigger && typeof ga !== 'undefined' && ga !== null) {
```

Comment:
`typeof ga !== 'undefined' && ga !== null`.... If this stays as a thing, maybe we turn into a helper ga function to call that returns a boolean so we're not repeating this code in places :)